lua/telescope/config.lua

```diff
@@ -37,7 +37,15 @@ function config.set_defaults(defaults)
   set("scroll_strategy", "cycle")
 
   set("layout_strategy", "horizontal")
-  set("layout_defaults", {})
+  local layout_defaults_table = {
+    horizontal = {
+      mirror = false,
+    },
+    vertical = {
+      mirror = false,
+    },
+  }
+  set("layout_defaults", { layout_defaults_table })
```

Comment:
I don't think this really does anything -- since the defaults are false, they'll come out `falsey` when we check them later. I don't know that you need to do this.

Reply:
Oh interesting, I didn't even think about that. It does look like removing that change altogether still makes my updates work properly!